reactor-core/src/main/java/reactor/core/publisher/FluxCreate.java

```diff
@@ -103,7 +103,8 @@ public void subscribe(CoreSubscriber<? super T> actual) {
 
 	@Override
 	public Object scanUnsafe(Attr key) {
-		return null; //no particular key to be represented, still useful in hooks
+		if (key == Attr.RUN_STYLE) return Attr.RunStyle.SYNC;
```

Comment:
isn't it potentially `ASYNC` source?

Reply:
Fixed in commit [abd4003](https://github.com/reactor/reactor-core/pull/2123/commits/abd40036e30e787e47e07fc7461c90fbdb1102e0)